src/util/BitField.js

```diff
@@ -99,12 +93,13 @@ class BitField {
    * Gets an object mapping field names to a {@link boolean} indicating whether the
    * bit is available.
    * @param {...*} hasParams Additional parameters for the has method, if any
-   * @returns {Object}
+   * @returns {Object<string, boolean>}
    */
   serialize(...hasParams) {
-    const serialized = {};
-    for (const [flag, bit] of Object.entries(this.constructor.FLAGS)) serialized[flag] = this.has(bit, ...hasParams);
-    return serialized;
+    return Object.entries(this.constructor.FLAGS).reduce((serialized, [flag, bit]) => {
+      serialized[flag] = this.has(bit, ...hasParams);
+      return serialized;
+    }, {});
```

Comment:
[`Object.fromEntries`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/fromEntries) can also be used.